src/modules/Checkbox/Checkbox.js

```diff
@@ -1,5 +1,5 @@
 import cx from 'classnames'
-import _ from 'lodash/fp'
+import _ from 'lodash'
```

Comment:
@levithomason I removed usage of `lodash/fp` because I don't understand how I make can make this in FP style:

```js
_.invoke(this.props, 'onMouseDown', e, { ...this.props, checked: !!checked, indeterminate: !!indeterminate })
```